node/src/protocol/mod.rs

```diff
@@ -359,9 +356,9 @@ async fn get_my_participant(protocol: &MpcSignProtocol) -> Participant {
     participant_info.id.into()
 }
 
-fn get_contract_version(mpc_contract_account: &AccountId) -> Result<i64, std::num::ParseIntError> {
-    let mpc_contract_id = mpc_contract_account.as_str();
-    let parts: Vec<&str> = mpc_contract_id.split('.').collect();
-    let version_str = parts[0].trim_start_matches('v');
-    version_str.parse::<i64>()
+fn node_version() -> i64 {
+    env!("CARGO_PKG_VERSION")
```

Comment:
this is the version in 
```
fn node_version() -> i64 {

    env!("CARGO_PKG_VERSION")

        .split('.')

        .map(|s| s.parse::<i64>().unwrap())

        .fold(0, |acc, x| acc * 1000 + x)

}
```?

Reply:
@ppca I'm not sure what you are asking. Can you please clarify?

Reply:
Ah I pasted the wrong thing. I mean this is the node/ version?

Reply:
This function will give us an integer representation of the version specified in `Cargo.toml`.